### PR TITLE
Messages and model update

### PR DESCRIPTION
**Pull request by florinacho337**
- updated model
- created message routes
- created message controller

### CODE REVIEW

**Review by Bacter16 — CHANGES_REQUESTED** [review]
Where message for message table? :))

File: back_end/src/controllers/MessageController.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const { PrismaClient } = require('@prisma/client');
+const prisma = new PrismaClient();
+
+const createMessage = async (req, res) => {
+    try {
+        const { text, userId, eventId } = req.body;
+        const message = await prisma.message.create({
+            data: {
+                text,
+                userId,
```
Comment (Domynis) [inline back_end/src/controllers/MessageController.js:10]: de pus senderId ca si in schema

**Review by Domynis — CHANGES_REQUESTED** [review]
e ok dar astea 2 chestii trb rezolvate

File: back_end/prisma/schema.prisma [hunk 2]
```diff
@@ -85,3 +90,28 @@ enum Sport {
   CARD_GAMES
   OTHER
 }
+
+model Message {
+  id        String   @id @default(uuid())
+  sender    User     @relation("SentMessages", fields: [senderId], references: [id])
```
[the remark this answers is not in the document]
Comment (florinacho337) [reply back_end/prisma/schema.prisma:96]: deci la un moment aveam textul la mesaj, dar nu stiu unde a disparut :))

**Review by Bacter16 — APPROVED** [review]

**Review by Tiberiw — APPROVED** [review]

**Pull request merged**